release.py

```diff
@@ -358,38 +357,37 @@ def pypi_release(opts, pargs):
     version = pargs[1]
 
     db_warning = "Warning: No new distribution build. This occurs when there are no new changes to the source code since the previous release. Please check for any untracked changes and update your package changelog/release-history to reflect the newest version."
-
-    # Only upload source
-    if opts.build_no_wheel:
-        call("python -m build --sdist", release_dir)
+    build_wheel = not opts.no_wheel
+    # Build wheel and source
+    if build_wheel:
+        call("python -m build", release_dir) 
         # Upload using twine
         no_tar = True
+        no_whl = True
         for file in list((release_dir / "dist").iterdir()):
             if re.search(f".*{version}.*.tar.gz", file.name):
                 no_tar = False
+            if re.search(f".*{version}.*.whl", file.name):
+                no_whl = False
         if no_tar:
             call(f"echo \"{db_warning}\"", release_dir)
+        elif no_whl:
+            call(f"echo \"Warning: No wheel found.\"", release_dir)
```

Comment:
let's take the chance to make the error messages more helpful.  Always "what went wrong.  suggestions to fix it".

I think this is good for now, but maybe let's make an issue to make the script check everything before doing anything and jumping out if something is wrong.  I have spent too much time having releases fail and then having to remake the news and clean out the changelog and delete the release on gh and so on.  A workflow where it doesn't do those things until it is more or less sure it will work would be super-handy.  Maybe we can give this project to a new student or sthg?  For now, just make an issue?